packages/server/src/services/bitquery/fetcher.ts

```diff
@@ -30,7 +30,24 @@ const authMiddleware = new ApolloLink((operation, forward) => {
     return forward(operation);
 })
 
-const cache = new InMemoryCache();
+const cache = new InMemoryCache({
+    typePolicies: {
+        Ethereum: {
+            merge: false,
```

Comment:
this fixes the apollo client error due to in memory cache trying to merge the Ethereum object, but it can't since there is no id